lib/assets/stylesheets/modules/_technical-documentation.scss

```diff
@@ -141,6 +141,8 @@
     font-family: monaco, Consolas, "Lucida Console", monospace;
     font-size: 15px;
     font-size: 0.9375rem;
+    // Match the line-height outside of tables
+    line-height: 1.4;
```

Comment:
I wonder if the keyword `normal` would work, but happy with this if not.

Reply:
I tested this and it would work for all the monospaced fonts we specify but the padding on the top and bottom of the line still causes some vertical overlap.